spec/features/heroku_spec.rb

```diff
@@ -4,8 +4,9 @@
   scenario "Suspend a project for Heroku" do
     run_suspenders("--heroku=true")
 
-    expect(FakeHeroku).
-      to have_gem_included(project_path, "rails_stdout_logging")
+    %w(rails_stdout_logging).each do |gem|
```

Comment:
Yeah I believe if it's too wide Hound would prefer:

```ruby
expect(FakeHeroku)
  .to have_gem_included(project_path, "rails_stdout_logging")
```

While it's only one gem I'd do it this way.